identity-aware-proxy/IAPClientTest/runTest.ps1

```diff
@@ -12,5 +12,5 @@
 # License for the specific language governing permissions and limitations under
 # the License.
 
-# Skipping because of: https://github.com/GoogleCloudPlatform/dotnet-docs-samples/issues/1040
-# dotnet test --test-adapter-path:. --logger:junit
\ No newline at end of file
+dotnet restore

```

Comment:
I don't think this is necessary. (It used to be, a long time ago.) Just running dotnet test should be fine.

Reply:
Done